common/test/test_snapshots.py

```diff
@@ -342,6 +344,9 @@ def test_change_owner_without_root(self):
         self.assertEqual(s.st_gid, CURRENTGID)
 
     @unittest.skipIf(NO_GROUPS, "Current user is in no other group. So this test won't work.")
+    @unittest.skipIf(ON_TRAVIS, "File change group commands don't work in travis")
+    # TODO - Find way to run travis-specified Python version when using settings to allow changing groups
+    # http://stackoverflow.com/questions/35117270/how-can-i-change-the-group-of-a-file-when-executing-in-a-travis-ci-build
```

Comment:
Turns out that getting travis to give the process running unit tests change group ability (required for this test) while maintaining the virtualenv for the specific python versions we want during the test is a bit tricky. I've got a TODO in there for fixing that along with a stackoverflow question. Maybe you have some bright idea? Until then, we can still run that specific test locally outside of travis.